otx/algorithms/anomaly/configs/base/configuration.py

```diff
@@ -19,6 +19,7 @@
 from attr import attrs
 
 from otx.algorithms.anomaly.configs.base.configuration_enums import (
+    ModelBackbone,
```

Comment:
I guess we don't need to import this if it's not used

Reply:
I'll take care. Thanks!